### ISSUE TITLE
Remove posts from NSFW subreddits

### ISSUE THREAD

**Issue by pmdevita** [pmdevita/switcharoohelper#16]
As per r/switcharoo rules, not a difficult addition

**Comment by gautam678** [comment]
I can take this issue.

**Comment by gautam678** [comment]
@pmdevita  How do your usually test a feature?

I've made a PR but I don't know how to test my feature.

https://github.com/pmdevita/switcharoohelper/pull/20

**Comment by pmdevita** [comment]
Yep thanks! Testing is difficult, you can turn it on a mode in the config that just makes it talk about what it would have done and then run it through the last 100 posts or so. I have started working on some unit tests and I already have tests for the regex but full testing for the bot would require the creation of one, possibly two, test subreddits which would then have to be filled with a lot of different kinds of tests. It would be a pretty significant undertaking so I haven't attempted it. However, now that I think about it, it would be handy to have to train new mods as well as test the bot.

I'm not at home right now and while I sync the project files between my desktop and laptop, I don't sync the local git repo and so it's difficult to do git operations at the moment without creating merge conflicts with the repo on my desktop. I'll probably be back by June and then I'll push in all my more recent stuff and a credentials.ini example and pull your patch down and test it. It should be fine from what I can tell though.

**Comment by pmdevita** [comment]
Here's the instructions for whoever picks this up.

A new issue should be created and appended to the master list. Then in the check_errors function, get the Subreddit object from the created PRAW Comment object and check whether it is `over_18`. This check should only be applied to roos newer than March 1st, 2021 due to some concerns with older roos. Then, strings should be added to the ModIssueStrings classes. Finally, a test case should be created as a part of the process/check_errors test cases
